BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -22,6 +22,11 @@
 #import "BOXUserRequest.h"
 #import "BOXContentClient+User.h"
 
+// Default API URLs
+NSString *const BOXDefaultAPIBaseURL = @"https://api.box.com/2.0";
```

Comment:
Do you think that including the version number could cause confusion. You'd have to know to set the base URL with the version number (this would have to be well documented, I can see this being easy to miss even for our own use).

Then this becomes a problem when the version number is not desirable... is this really the true base URL?
Should we split it to baseURL and baseURLWithVersion?